GoogleForCreators/web-stories-wp
# Enable Media library video optimization by default

## Context



## Summary

Remove the feature flag for video optimization via media library. 

## Relevant Technical Choices

Move `toggleVideoOptimization` util to e2e-utils package.
Tweak e2e tests to use toggleVideoOptimization util 
Remove `enableMediaPickerVideoOptimization` feature flag.

## To-do



## User-facing changes



## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Create new story. 
1. Click upload button. 
1. Upload, .mov or .mpg file. 
1. Wait for file to upload. 
1. Once complete, click insert in to page. 
1. See video is inserted. 
1. See placeholder is inserted.
1. See update once poster is generated, element may change size and position. 
1. See that video controls are not visible. 
1. Once trancode is finished, video should appear in sidebar and video should be playable. 

This functionality should be work, if users disables transcoding options. See dashboard -> settings. 

1. Go to media in sidebar. 
1. Upload .mov or .mpg file by clicking "add new"
1. Create new story. 
1. Click upload button. 
1. Click on media library. 
1. Once complete, click insert in to page. 
1. See video is inserted. 
1. See placeholder is inserted.
1. See update once poster is generated, element may change size and position. 
1. See that video controls are not visible. 
1. Once trancode is finished, video should appear in sidebar and video should be playable. 

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #7842
